packages/framework/package.json

```
@@ -26,11 +26,12 @@
     "format:fix": "prettier --write --ignore-path .gitignore .",
     "build": "tsup && pnpm check:circulars",
```

We should remove the addition invocation of `check:circulars` in `build` now that we have `postbuild`
```suggestion
    "build": "tsup",
```